lib/Serverless.js

```diff
@@ -70,6 +71,12 @@ class Serverless {
       return this.pluginManager.run(this.processedInput.commands);
     }
 
+    // check if tracking is enabled (and track if it's enabled)
+    const serverlessPath = this.config.serverlessPath;
```

Comment:
Just to make sure ServerlessPath is the directory that serverless was installed into right? So we're not tracking this in the home directory, but in some install path. Fine with me, the only issue this could be is if Serverless is installed somewhere we don't have write access to. Then this might simply fail and kill the whole process. Might want to put an exception handling around it and not track if we can't read the do-not-track path